sema4ai/vscode-client/src/robo/actionPackage.ts

```diff
@@ -1066,7 +1066,7 @@ const getActionPackageStatus = async (
         }
     );
     if (!result.success) {
-        window.showErrorMessage(`Failed to get action package status: ${result.message}`);
+        showErrorMessageWithShowOutputButton("Failed to get action package status");
```

Comment:
Question (just double-checking): Have you checked that the `result.message` is actually added to the OUTPUT_CHANNEL (in all the scenarios where it was removed)?

Reply:
yes, on all